lib/OpenQA/Command.pm

```diff
@@ -119,6 +119,7 @@ sub url_for ($self, $path) {
 }
 
 sub retry_tx ($self, $client, $tx, $retries = undef, $delay = undef) {
+    local $ENV{MOJO_CONNECT_TIMEOUT} = 30;
```

Comment:
This might also be a decrease, e.g. if the user had explicitly exported `MOJO_CONNECT_TIMEOUT=60`.

```suggestion
    local $ENV{MOJO_CONNECT_TIMEOUT} = 30 unless defined $ENV{MOJO_CONNECT_TIMEOUT};
```

Or maybe just (but I'm not sure how it'll behave together with `local`):
```suggestion
    local $ENV{MOJO_CONNECT_TIMEOUT} //= 30;
```

Reply:
done. `//=` works with `local`